src/domain/collaboration/callout/callout.service.ts

```diff
@@ -450,7 +450,7 @@ export class CalloutService {
     shuffle?: boolean
   ): Promise<IAspect[]> {
     const loadedCallout = await this.getCalloutOrFail(callout.id, {
-      relations: ['aspects'],
+      relations: ['aspects', 'aspects.comments'],
```

Comment:
comments are not needed for the impl of this method, a use-case is requiring the comments to be loaded